plugin/trino-mysql-event-listener/src/main/java/io/trino/plugin/eventlistener/mysql/MysqlEventListenerConfig.java

```diff
@@ -26,6 +27,7 @@ public String getUrl()
         return url;
     }
 
+    @ConfigSecuritySensitive
```

Comment:
Do we need to add verification like we handle for jdbc url ? In case of debugging it would be helpful

Reply:
Added a validated logic with tests.